features/step_definitions/user_steps.rb

```diff
@@ -177,6 +177,11 @@
   user.roles = [Role.find_or_create_by(name: "protected_user")]
 end
 
+Given "the user {string} is hidden from search engines" do |login|
```

Comment:
This should probably go in `features/step_definitions/preferences_steps.rb` instead. The tests themselves could probably also be moved to `features/other_a/preferences_edit_more.feature`.